client/components/ReportsPanel/ReportsTabbar.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component, PropTypes } from 'react';
+
+import './style.scss';
+
+const Link = props => (
```

Comment:
Wouldn't we want Link to be it's own export in its own file? ...in case we want to use it somewhere else?